packages/cozy-harvest-lib/src/components/TwoFAModal.jsx

```diff
@@ -21,8 +21,11 @@ export class TwoFAForm extends PureComponent {
     this.setState({ twoFACode: e.currentTarget.value })
   }
 
-  handleSubmit() {
+  handleSubmit(e) {
     this.props.konnectorJob.sendTwoFACode(this.state.twoFACode)
+    // prevent refreshing the page when submitting
+    // (happens not systematically)
+    e.preventDefault()
```

Comment:
I think calling `e.preventDefault()` as first instruction is a good convention (At least a personal convention), could we do this here ?

Reply:
Also I find comments like `     // (happens not systematically) ` not very useful, it is fear and doubt inducing.

Reply:
Keeping just `prevent refreshing the page when submitting` means that the issue happens systematically which is not the case here and we could remove this part thinking that we don't need it anymore due to a false positive